### PR TITLE
Added inline-documentation for funcs in eval.go

### PR DESCRIPTION
**Pull request by sahil1913**
Resolves: #29

### CODE REVIEW

File: core/eval.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ func init() {
 	txnCommands = map[string]bool{"EXEC": true, "DISCARD": true}
 }
 
+// Returns the status of the connection with a "PONG"
```
Comment (arpitbbhayani) [inline core/eval.go:26]: "returns the status of the connection" seems wrong. We are simply responding with `PONG`. Let's change the language here.

File: core/eval.go [hunk 2]
```diff
@@ -39,6 +40,8 @@ func evalPING(args []string) []byte {
 	return b
 }
 
+// Adds new entry for the specifid key-value pair
+// as per the sent expiry time.
```
Comment (arpitbbhayani) [inline core/eval.go:44]: Here are some guidelines for writing good comments. https://tip.golang.org/doc/comment would recommend going through this and rewriting. Let's be more verbose.

File: core/eval.go [hunk 3]
```diff
@@ -97,6 +102,9 @@ func evalGET(args []string) []byte {
 	return Encode(obj.Value, false)
 }
 
+// Returns the Time-to-Live for the specified key if it exists
+// Returns -1 in case no expiration is set on the key
+// Returns -2 in case key doesn't exist
```
Comment (arpitbbhayani) [inline core/eval.go:107]: Every command needs to have a standard verbose comment. The format would help us standardize code commenting across the repository. The comment we add in this file will lay the foundation.

File: core/eval.go [hunk 4]
```diff
@@ -249,6 +270,9 @@ func evalSLEEP(args []string) []byte {
 	return RESP_OK
 }
 
+// Marks the start of the transaction.
+// All subsequent commands will be queued for 
+// atomic execution using EXEC
```
Comment (arpitbbhayani) [inline core/eval.go:275]: Review comments mentioned above applies to all.

**Review by arpitbbhayani — CHANGES_REQUESTED** [review]

**Comment by arpitbbhayani** [comment]
@sahil1913 Can you please update the PR?

File: core/eval.go (hunk 1 above)
Comment (sahil1913) [reply core/eval.go]: updated.

File: core/eval.go (hunk 2 above)
Comment (sahil1913) [reply core/eval.go]: sure thanks for sharing. Rewrote the comments as suggested.

**Review by arpitbbhayani — APPROVED** [review]

**Comment by arpitbbhayani** [comment]
Super. Looks great. Thanks a ton @sahil1913

**Pull request merged**